src/01-functions.js

```diff
@@ -112,7 +132,16 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
```

Comment:
Since our if else statement returns true or false, we can just return our comparison (`return a === b`), as it will evaluate to a boolean.